src/main/java/com/hrznstudio/titanium/block/tile/ActiveTile.java

```diff
@@ -66,8 +69,8 @@
 import java.util.List;
 
 public abstract class ActiveTile<T extends ActiveTile<T>> extends BasicTile<T> implements IScreenAddonProvider,
-        ITickableTileEntity, INamedContainerProvider, IButtonHandler, IFacingComponentHarness, IContainerAddonProvider,
-        IHasAssetProvider {
+    ITickableTileEntity, INamedContainerProvider, IButtonHandler, IFacingComponentHarness, IContainerAddonProvider,
+    IHasAssetProvider, IRedstoneReader {
```

Comment:
IMO, this shouldn't be forced on ActiveTile, it's extra code just attached to the object.
I think personally this should be something each author can chose to add.

I'd say attach it rather on PoweredTile or create a RedstoneTile and make Powered extend that?